src/v3/stylis-logical-plugin/src/utils/ensmallen.ts

```diff
@@ -0,0 +1,5 @@
+import { compile, serialize, stringify } from 'stylis';
+
+export const ensmallen = (css: string) => {
```

Comment:
Unusual name. Is this borrowed?

Reply:
The opposite of [embiggen](https://en.wikipedia.org/wiki/Lisa_the_Iconoclast#Embiggen_and_cromulent), a perfectly cromulent word.

Reply:
I can rename this though, I was just writing a little util to help with testing

Reply:
Are you handling the renames in this PR?

Reply:
yeah, will rename before merging